library/src/main/java/com/getstream/sdk/chat/navigation/destinations/AttachmentDestination.java

```diff
@@ -121,21 +122,25 @@ private void loadFile(Attachment attachment) {
         String mimeType = attachment.getMime_type();
         String url = attachment.getAssetURL();
 
-        // Media
-        if (mimeType.contains("audio") ||
-                mimeType.contains("video")) {
-            Intent intent = new Intent(context, AttachmentMediaActivity.class);
-            intent.putExtra("mimeType", mimeType);
-            intent.putExtra("url", url);
-            start(intent);
-        } else if (mimeType.equals("application/msword") ||
-                mimeType.equals(ModelType.attach_mime_txt) ||
-                mimeType.equals(ModelType.attach_mime_pdf) ||
-                mimeType.contains("application/vnd")) {
-
-            Intent intent = new Intent(context, AttachmentDocumentActivity.class);
-            intent.putExtra("url", url);
-            start(intent);
+        if (mimeType == null) {
+            StreamChat.getLogger().logE(this, "MimeType is null");
```

Comment:
We should react somehow, otherwise user will see nothing when he tries to open attachment. Let's add at least `Toast` with a message like "Something went wrong. Unable to open attachment":
```kotlin
Utils.showMessage(context, context.getString(R.string.stream_attachment_invalid_mime_type));
```